backend/core/scripts/import-unit-groups-basic.ts

```diff
@@ -0,0 +1,166 @@
+import * as fs from "fs"
+import CsvReadableStream from "csv-reader"
+import { Connection, DeepPartial } from "typeorm"
+import Listing from "../src/listings/entities/listing.entity"
+import { Jurisdiction } from "../src/jurisdictions/entities/jurisdiction.entity"
+import dbOptions = require("../ormconfig")
+import { Program } from "../src/program/entities/program.entity"
+import { AddressCreateDto } from "../src/shared/dto/address.dto"
+
+// TODO the general question is what is the current state of the DB this script is targeting
+//  probably programs do not exist in the DB, so are not searchable by title as it is done in this script
+// TODO how to handle Additional Accessibility
+// TODO is hearing/mobile/visual a unit_accessibility_priority_types ? Also this s a property of a unit, there are no units defined in this spreadsheet
```

Comment:
@pbn4 ,

1. as discussed in planning, this will be run on a db without listings and/or we don't care about any existing listings. Programs are getting added via [this pr](https://github.com/CityOfDetroit/bloom/pull/1022)
2. goes into one of the text fields. @kathyccheng , is it special notes?
3. those are to get added to listingFeatures